content/docs/user-guide/pipelines/run-cache.md

```diff
@@ -3,18 +3,20 @@
 Every time you [reproduce](/doc/command-reference/repro) a pipeline with DVC, it
 logs the unique signature of each stage run (in `.dvc/cache/runs` by default).
 If it never happened before, its command(s) are executed normally. Every
-subsequent time a [stage](/doc/command-reference/stage/add) runs under the same
-conditions, the previous results can be restored instantly, without wasting time
-or computing resources.
+subsequent time a <abbr>stage<abbr> runs under the same conditions, the previous
+results can be restored instantly -- without wasting time or computing
+resources.
+[More details](/doc/user-guide/project-structure/internal-files#run-cache)
+
+✅ This built-in feature is called **run cache** and it can dramatically improve
+performance. It's enabled out-of-the-box (can be disabled), which means DVC is
+already saving all of your tests and experiment results behind the scene.
 
 <admon type="warning">
 
-If any output of a stage has `cache: false`, the <abbr>run-cache</abbr> will be
-deactivated for that stage
+If an output of a stage has `cache: false`, the run cache will be deactivated
+for that stage.
 
 </admon>
 
-✅ This built-in feature is called <abbr>run-cache</abbr> and it can
-dramatically improve performance. It's enabled out-of-the-box (can be disabled),
-which means DVC is already saving all of your tests and experiments behind the
-scene.
+See also [DVC Experiments](/doc/user-guide/experiment-management).
```

Comment:
```suggestion
The run cache is also enabled when you use `dvc exp run`.
```

What do you think about being more explicit here to explain how it relates to experiments?